### ISSUE TITLE
ascent slices do not work on GPU (likely ascent bug)

### ISSUE THREAD

**Issue by BenWibking** [parthenon-hpc-lab/athenapk#39]
The Ascent outputs do not work on GPU for the precipitator pgen. It works fine on host runs.

On GPU runs, we instead get:
```
s1/p1 pseudocolor plot yielded no data, i.e., no cells remains2/p1 pseudocolor plot yielded no data, i.e., no cells remaincycle=1465 time=2.6308698410348811e+03 dt=1.7657586286923035e+00 zone-cycles/wsec_step=4.56e+05 wsec_total=1.23e+03 wsec_step=1.15e+00
```
and

![dens00000](https://user-images.githubusercontent.com/1781919/225992184-57085ca2-d8c3-48b7-befd-4ad511675536.png)

**Comment by BenWibking** [comment]
This works:
```
-
  action: "add_scenes"
  scenes:
    s1:
      plots:
        p1:
          type: "pseudocolor"
          field: "Density"
          color_table:
            name: "Viridis"
      renders:
        r1:
          image_prefix: "dens%05d"
          annotations: "true"
          camera:
            look_at: [0.0, 0.0, 0.0]
            position: [0.0, -200.0, 0.0]
            up: [0.0, 0.0, 1.0]
```
while this does not:
```
-
  action: "add_pipelines"
  pipelines:
    pl1:
      f2:
        type: "slice"
        params:
          point:
            x: 0.0
            y: 0.0
            z: 0.0
          normal:
            x: 0.0
            y: 1.0
            z: 0.0
-
  action: "add_scenes"
  scenes:
    s1:
      plots:
        p1:
          type: "pseudocolor"
          field: "Density"
          pipeline: "pl1"
          color_table:
            name: "Viridis"
      renders:
        r1:
          image_prefix: "dens%05d"
          annotations: "true"
          camera:
            look_at: [0.0, 0.0, 0.0]
            position: [0.0, -200.0, 0.0]
            up: [0.0, 0.0, 1.0]
```

**Comment by BenWibking** [comment]
I've opened an Ascent issue here: https://github.com/Alpine-DAV/ascent/issues/1119

**Comment by BenWibking** [comment]
Fixed in VTKm-2.1rc2.
